File: Python/Program/Character.py
class Character:
    def __init__(self, name: str, lives_at: str, main_skill: str):
        self._name = name
        self._lives_at = lives_at
        self._main_skill = main_skill

    def get_name(self) -> str:
        return self._name

    def set_name(self, name: str):
        self._name = name

    def get_lives_at(self) -> str:
        return self._lives_at

    def set_lives_at(self, lives_at: str):
        self._lives_at = lives_at

    def get_main_skill(self) -> str:
        return self._main_skill

    def set_main_skill(self, main_skill: str):
        self._main_skill = main_skill

File: C++/Program/Character.cpp
#pragma once
#include <iostream>
#include <string>

using namespace std;

class Character {
private:
    string name;
    string livesAt;
    string mainSkill;

public:
    // Constructor
    Character(){}

    Character(string name, string livesAt, string mainSkill) {
        this->name = name;
        this->livesAt = livesAt;
        this->mainSkill = mainSkill;
    }

    // Getter and Setter for name
    string getName() {
        return name;
    }

    void setName(string name) {
        this->name = name;
    }

    // Getter and Setter for livesAt
    string getLivesAt() {
        return livesAt;
    }

    void setLivesAt(string livesAt) {
        this->livesAt = livesAt;
    }

    // Getter and Setter for mainSkill
    string getMainSkill() {
        return mainSkill;
    }

    void setMainSkill(string mainSkill) {
        this->mainSkill = mainSkill;
    }
};